### PR TITLE
Creating new event page

### PR DESCRIPTION
**Pull request by tiagolimpe**
<p><b>Feature/Bug description:</b> The event page are out of sync with new design.</p>

screenshot from 2018-01-10 11-16-53

<p><b>Solution:</b> Changing html and controllers to make the new event page.</p>

event-page

<p><b>TODO/FIXME:</b> 

- Adjust the way to define start_time and end_time of the event, because an event can happen in a sequence of days with pauses (in a night to the next day, for example).

- Define what is the official site, because in creation of event not have this field and the event in backend not have this property.</p>

### CODE REVIEW

File: frontend/event/event_details.html [hunk 1]
```diff
@@ -4,31 +4,149 @@
...
+              <h4 style="color: #03b3a1;">Horário</h4>
+              <div layout="row" class="left-justify">
+                <p>
+                  <md-icon style="font-size: 40px;">access_time</md-icon>
+                </p>
+                <div layout="column" style="margin-left: 40px;">
+                  <h3>{{eventDetailsCtrl.event.start_time | amUtc | amLocal | amDateFormat:'H:mm' | uppercase}} às
+                      {{eventDetailsCtrl.event.end_time | amUtc | amLocal | amDateFormat:'H:mm' | uppercase}}
+                  </h3>
+                  <span style="margin-top: -15px; color: #959796;">Carga horária: não definida</span>
```
Comment (mayzabeel) [inline frontend/event/event_details.html]: Coloca no TODO do pull request que precisa disso no backend

File: frontend/event/event_details.html [hunk 2]
```diff
@@ -4,31 +4,149 @@
...
+                </div>
+              </div>
+            </div>
+            <div layout="column" style="margin-left: 200px;">
+              <h4 style="color: #03b3a1;">Site oficial</h4>
```
Comment (mayzabeel) [inline frontend/event/event_details.html]: coloca no TODO também

File: frontend/event/eventController.js [hunk 3]
```diff
@@ -95,6 +95,7 @@
             promise.then(function success() {
                 eventCtrl.events = eventCtrl.events.filter(thisEvent => thisEvent.key  !== event.key);
                 MessageService.showToast('Evento removido com sucesso!');
+                if (eventCtrl.isEventPage) $state.go('app.user.events');
```
Comment (andreldsa) [inline frontend/event/eventController.js]: This verification is very confusing, why you have a flag to ensure a state that should be linked directly to a controller?

File: frontend/event/eventController.js [hunk 4]
```diff
@@ -140,18 +141,28 @@
...
-            $state.go('app.event', {eventKey: event.key});
+            $state.go('app.user.event', {eventKey: event.key});
         };
 
 
         eventCtrl.endInOtherMonth = function endInOtherMonth() {
-            const startMonth = new Date(eventCtrl.event.start_time).getMonth();
-            const endMonth = new Date(eventCtrl.event.end_time).getMonth();
-            return startMonth !== endMonth;
+            if(eventCtrl.event) {
```
Comment (andreldsa) [inline frontend/event/eventController.js]: test

File: frontend/event/eventController.js [hunk 5]
```diff
@@ -140,18 +141,28 @@
...
+                const startMonth = new Date(eventCtrl.event.start_time).getMonth();
+                const endMonth = new Date(eventCtrl.event.end_time).getMonth();
+                return startMonth !== endMonth;
+            }
+        };
+
+        eventCtrl.getVideoUrl = function getVideoUrl(video_url) {
```
Comment (andreldsa) [inline frontend/event/eventController.js]: test

**Pull request merged**